bin/zeppelin-daemon.sh

```diff
@@ -94,7 +94,8 @@ function initialize_default_directories() {
     $(mkdir -p "${ZEPPELIN_PID_DIR}")
   fi
 
-  echo "Browse localhost:8080 in your browser"
+  IP_ADDR="$(ifconfig en0 | grep inet | grep -v inet6 | cut -d ' ' -f2)"
+  echo -e "Browse $IP_ADDR:8080 in your browser.\nIf you are testing on your local computer, use localhost:8080"
```

Comment:
@djKooks As you can see in the referred docs
>You can configure Apache Zeppelin with either environment variables in conf/zeppelin-env.sh (conf\zeppelin-env.cmd for Windows) or Java properties in conf/zeppelin-site.xml. If both are defined, then the environment variables will take priority. 

User can change the port using both `zeppelin-site.xml` and `zeppelin-env.sh`. If the user modify only `zeppelin-env.sh` or both, then you can easily refer `zeppelin-env.sh` file. But we should also cover the case that only `zeppelin-site.xml` is changed.